### PR TITLE
Bugfixes2

### PR DESCRIPTION
**Pull request by au1994**
@vaidik
@vishesh92
@dash1291

### CODE REVIEW

File: polls/imdbScraper.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    except requests.exceptions.RequestException as e:
+        logging.info(e)
+        traceback.print_exc(e)
+
+    soup = BeautifulSoup(r.content, "html.parser")
+    movie_details = soup.find("div", {"class": "plot_summary"})
+    json_array = []
+
+    if movie_details is None:
+        print "No results found"
```
Comment (dash1291) [inline polls/imdbScraper.py]: `print` 👎

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Django == 1.9.2
+BeautifulSoup4
+Requests
+gunicorn == 19.4.5
+whitenoise == 2.0.6
```
Comment (dash1291) [inline requirements.txt:5]: whitenoise? what is it exactly?

Comment (au1994) [inline requirements.txt:5]: Used for serving static files in python web apps

File: mysite/settings.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    }
+}
+
+
+# Password validation
+# https://docs.djangoproject.com/en/1.9/ref/settings/#auth-password-validators
+
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
```
Comment (vishesh92) [inline mysite/settings.py:91]: Line length?

File: polls/views.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    page = request.GET.get('page')
+
+    try:
+        movies = paginator.page(page)
+    except PageNotAnInteger:
+        movies = paginator.page(1)
+    except EmptyPage:
+        movies = paginator.page(1)
+
+    # print type(movies)
```
Comment (vishesh92) [inline polls/views.py]: Remove commented code.

File: polls/mylib.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+def printname():
```
Comment (vishesh92) [inline polls/mylib.py]: Is this required?

File: polls/imdbScraper.py [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+        json_object['url'] = url
+        json_object['id'] = movie_id
+        json_array.append(json_object)
+
+    return json_array
+
+
+def get_movie_results(movie_id):
+
+    url = "http://www.imdb.com/title/" + movie_id + "/?ref_=fn_al_tt_1"
```
Comment (vishesh92) [inline polls/imdbScraper.py]: I don't think you need this part `"/?ref_=fn_al_tt_1"`

File: polls/imdbScraper.py [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+
+def get_search_results(movie):
+
+    url = "http://www.imdb.com/find?s=tt&q=" + movie
+
+    try:
+        r = requests.get(url)
+    except requests.exceptions.RequestException as e:
+        logging.info(e)
+        traceback.print_exc(e)
```
Comment (vishesh92) [inline polls/imdbScraper.py:18]: You should return something here.

File: polls/mylib.py (hunk 5 above)
Comment (au1994) [inline polls/mylib.py]: no just for testing

File: polls/imdbScraper.py [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+        json_object = {}
+        json_object['error'] = "Service unavailable"
+        return json_object
```
Comment (vishesh92) [inline polls/imdbScraper.py]: What will be the status code?

Comment (au1994) [inline polls/imdbScraper.py]: Status code has been handled in views. Please turn off wifi and then try searching a movie on localhost.

**Pull request merged**